File: PRAKTIKUM_4/4/PRAK404-2410817110010-DanielNoprianto.c
#include <stdio.h>

int main ()
{
    int pilihan;
    float nilai1, nilai2, hasil;

    while (1){
    printf("Pilih program\n");
    printf("1. Penjumlahan\n");
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. Exit\n");

    printf("Masukkan Pilihan : ");
    scanf("%d", &pilihan);
    
    if (pilihan == 5){
        printf("Terimakasih, telah menggunakan kalkulator DANIEL NOPRIANTO");
        break;
    }
    
    if (pilihan < 1||pilihan > 5){
        printf("Input Anda salah, silahkan coba lagi\n\n");
        continue;
    }
    
        switch (pilihan){
        case 1:
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilai1);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilai2);
            hasil = nilai1 + nilai2;
            printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);
            break;

        case 2:
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilai1);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilai2);
            hasil = nilai1 - nilai2;
            printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);
            break;

        case 3:
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilai1);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilai2);
            hasil = nilai1 * nilai2;
            printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);
            break;

        case 4:
            printf("Masukkan nilai pertama : ");
            scanf("%f", &nilai1);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &nilai2);
            hasil = nilai1 / nilai2;
            printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);
            break;
        default:
            break;
        }
    }
    return 0;
}

File: PRAKTIKUM_4/4/PRAK404-2410817110010-DanielNoprianto.py
while True:
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    pilihan = int(input("Masukkan pilihan : "))

    if pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator DANIEL NOPRIANTO")
        break

    elif pilihan >= 1 and pilihan <= 5:

        nilai_pertama = float(input("Masukkan nilai pertama : "))
        nilai_kedua = float(input("Masukkan nilai pertama : "))

        if pilihan == 1:
            hasil = nilai_pertama + nilai_kedua
            print(f"Hasil perjumlahan antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")
            print("")

        elif pilihan == 2:
            hasil = nilai_pertama - nilai_kedua
            print(f"Hasil pengurangan antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")
            print("")

        elif pilihan == 3:
            hasil = nilai_pertama * nilai_kedua
            print(f"Hasil perkalian antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")
            print("")

        elif pilihan == 4:
            hasil = nilai_pertama / nilai_kedua
            print(f"Hasil pembagian antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")
            print("")
        
    else:
        print("Input anda salah, silahkan coba lagi")
        print("")
        continue